app/src/main/java/com/globant/activities/CharactersActivity.kt

```diff
@@ -27,16 +32,23 @@ class CharactersActivity : AppCompatActivity() {
     private fun updateUI(charactersData: Event<Data<List<MarvelCharacter>>>) {
         when (charactersData.peekContent().responseType) {
             Status.GetCharacterError -> {
-                progress.visibility = View.GONE
+                progress_bar.visibility = View.GONE
                 charactersData.peekContent().error?.message?.let {
                     Toast.makeText(this, it, Toast.LENGTH_LONG).show()
                 }
             }
             Status.LOADING -> {
-                progress.visibility = View.VISIBLE
+                progress_bar.visibility = View.VISIBLE
             }
             Status.GetCharacterSuccess -> {
-                progress.visibility = View.GONE
+                progress_bar.visibility = View.GONE
+                charactersData.peekContent().data?.let {
+                    adapter = CharacterAdapter(it) {
+
+                    }
```

Comment:
do you have any TODO here?

Reply:
No, in next commit i will add a TODO comment